src/Tribe/Service_Providers/Context.php

```diff
@@ -284,6 +298,11 @@ static function ( $post_type ) {
 				],
 				'venue_post_type'      => [
 					'read' => [
+						Tribe__Context::FUNC          => [
+							static function () {
+								return ! empty( tribe_get_request_var( Venue::POSTTYPE, false ) );
+							}
```

Comment:
* :no_entry_sign: Each array item in a multi-line array declaration must end in a comma
* :no_entry_sign: Comma required after last value in array declaration

<sub>hash: 62810cdbb2773eee3368931a3ca0b158</sub>